firmware/makefile

```diff
@@ -265,7 +265,7 @@ all: $(FW_FILE1) $(FW_FILE2) $(ASSETS_FILE)
 
 # To build the two firmware files for flashing, you have to build the main file
 $(FW_FILE1) $(FW_FILE2) : $(FW_FILE0)
-	$(ESP_ROOT)/esptool/esptool.py elf2image $(FW_FILE0)
+	PATH=$(PATH):$(GCC_FOLDER)/bin python2 $(ESP_ROOT)/esptool/esptool.py elf2image $(FW_FILE0)
```

Comment:
`python2` make sense, but why add the xtensa gcc folder to the path?

Reply:
Also might as well replace `python2 $(ESP_ROOT)/esptool/esptool.py` with `$(ESPTOOL_PY)`

Reply:
It's not natively in my path.  This doesn't seem to be an issue elsewhere but is needed.

Reply:
Ah, adding it to the path is [step 2.3 of the getting started guide](https://github.com/AEFeinstein/Super-2021-Swadge-FW-Sandbox/blob/master/docs/GETTING_STARTED.md), which is why I haven't run into an issue here yet. I'll merge the change, can't hurt.